pandas/tests/tseries/test_resample.py

```diff
@@ -2670,6 +2670,11 @@ def test_resample_weekly_bug_1726(self):
         # it works!
         df.resample('W-MON', closed='left', label='left').first()
 
+    def test_resample_tz_aware_bug_15549(self):
+        index = pd.DatetimeIndex([1450137600000000000, 1474059600000000000], tz='UTC').tz_convert('America/Chicago')
+        df = pd.DataFrame([1, 2], index=index)
+        df.resample('12h', closed='right', label='right').last().ffill()
```

Comment:
agree, I will fix it. I just used the same notation as in the test above. thought it's a common thing